libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -1712,22 +1757,26 @@ bool AP_AHRS_NavEKF::get_variances(float &velVar, float &posVar, float &hgtVar,
         return true;
 #endif
     }
+
+    return false;
 }
 
 void AP_AHRS_NavEKF::setTakeoffExpected(bool val)
```

Comment:
I don't understand why we're not telling both EKFs that takeoff is expected.

Reply:
Yes, I would expect that we should tell any active EKFs that we're expecting a takeoff.

Reply:
https://github.com/ArduPilot/ardupilot/pull/13045